src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let score = 1;
```

Comment:
Why 1? Start with 0 and add `score` after function is called. Therefore, you'll only write `score++` once